apex/contrib/csrc/group_norm/group_norm_nhwc.h

```diff
@@ -64,9 +64,15 @@ static inline __device__ void spin_wait_(int *barrier, int step, int expected) {
 ////////////////////////////////////////////////////////////////////////////////////////////////////
 
 enum PrecisionMode {
-  FP16,
-  BF16,
-  FP32,
+  FP32IOFP16W,
+  FP32IOBF16W,
+  FP32IOFP32W,
+  FP16IOFP16W,
+  FP16IOBF16W,
+  FP16IOFP32W,
+  BF16IOFP16W,
+  BF16IOBF16W,
+  BF16IOFP32W,
```

Comment:
could you leave a comment what the first and the second dtype mean?

Reply:
Sure, updated.